### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by brett0x0** [Qvin0000/ExileApi#30]
Game not found windows 10 1903 steamx64

**Comment by Veight** [comment]
This should be posted in the readme. It only works for the standalone version.

**Comment by zaafar** [comment]
yeah a lot needs to be posted in the readme file. Feel free to create a PR to add these stuff into the readme file.
